### ISSUE TITLE
scrap `ensemble_seg_images_in_folder`

### ISSUE THREAD

**Issue by dbuscombe-usgs** [dbuscombe-usgs/segmentation_zoo#55]
This script now appears to facilitate ensemble models. What is the purpose of this? We already have an `ensemble` version of the script

```

W=[]
W.append(weights)

result = 'yes'
while result == 'yes':
    result = messagebox.askquestion("More Weights files?", "More Weights files?", icon='warning')
    if result == 'yes':
        root = Tk()
        root.filename =  filedialog.askopenfilename(title = "Select weights file",filetypes = (("weights file","*.h5"),("all files","*.*")))
        weights = root.filename
        root.withdraw()
        W.append(weights)
```

this is confusing - it appears that we have 2 scripts that now do identical things?

**Comment by dbuscombe-usgs** [comment]
this involves:
1. removing `ensemble_seg_images_in_folder`
2. modify `do_seg` in `prediction_imports` to deal with npz files, and ND imagery, as well as jpegs for 1 or 3 bands
3. new function `seg_file2tensor_ND` that reads ND data from npz (instead of  `seg_file2tensor_3band` for 1 or 3 bands)
4. model.predict works the same (depends on how it is compiled)
5. not sure yet how this affects `TESTTIMEAUG` but I think `flipud` and `fliplr` work fine on >3D inputs

**Comment by dbuscombe-usgs** [comment]
also removing from `do_seg` the temperature to scale the logits ... that was an experimental remnant from the watermasking project and is no longer needed

**Comment by ebgoldstein** [comment]
np.flipud and np.fliplr both work on set axis, so they hsould be fine (and we will notice if the predictions being super bad 😆 )

**Comment by dbuscombe-usgs** [comment]
this is now implemented and tested on Hatteras 5D data. There is now a single `seg_images_in_folder` script that does ND segmentation, using either merged data as npz, or jpegs/pngs

**Comment by dbuscombe-usgs** [comment]
first implemented in 909f84827ab48c5498074c2a806c130436844cfb

needs further testing
